util/src/main/java/io/kubernetes/client/util/version/Version.java

```diff
@@ -0,0 +1,53 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package io.kubernetes.client.util.version;
+
+import io.kubernetes.client.openapi.ApiClient;
+import io.kubernetes.client.openapi.ApiException;
+import io.kubernetes.client.openapi.Configuration;
+import io.kubernetes.client.openapi.apis.VersionApi;
+import io.kubernetes.client.openapi.models.VersionInfo;
+import java.io.IOException;
+import okhttp3.Call;
+import okhttp3.Response;
+
+public class Version {
+
+  private final VersionApi versionApi;
+
+  public Version() {
+    this(Configuration.getDefaultApiClient());
+  }
+
+  public Version(ApiClient apiClient) {
+    this.versionApi = new VersionApi(apiClient);
+  }
+
+  public VersionInfo getVersion() throws ApiException, IOException {
```

Comment:
Does this actually ever throw IOException?

Reply:
It actually does but it's not able to reach K8 cluster for some reason. But I will need to handle this.

Reply:
Sorry i meant, the code to deserialize response will throw IOException.

Reply:
ah, ok.